CONTRIBUTING.rst

```diff
@@ -0,0 +1,104 @@
+.. _contributing:
+
+Contributing
+------------
+
+If you find errors, omissions, inconsistencies or other things that need
+improvement, please create an issue or a pull request at
+https://github.com/sfstoolbox/theory/.
+Contributions are always welcome!
+
+
+Installation
+^^^^^^^^^^^^
+
+In order to build the theory section locally you should get the
+newest development version from Github_ and install the needed dependencies
+with::
+
+   git clone https://github.com/sfstoolbox/theory.git
+   cd theory
+   python3 -m pip install --user -e .
+
+.. _Github: https://github.com/sfstoolbox/theory/
+
+This way, your installation always stays up-to-date, even if you pull new
+changes from the Github repository.
+
+
+Building the Documentation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you make changes to the documentation, you can re-create the HTML and PDF
+pages using Sphinx_.
+
+To create the HTML pages, use::
+
+   make html-preview
+
+The generated files will be available in the directory
+``build/sphinx/html-preview/``.
+To create the PDF pages, use::
+
+    make latexpdf
+
+The generated files will be available in the directory ``build/sphinx/latex/``.
+
+.. _Sphinx: http://sphinx-doc.org/
+
+
+Creating a New Release
+^^^^^^^^^^^^^^^^^^^^^^
+
+New releases are made using the following steps:
+
+#. Update ``NEWS.rst``
+#. Commit those changes as "Release x.y.z"
+#. Create an (annotated) tag with ``git tag -a x.y.z``
+#. Push the commit and the tag to Github and `add release notes`_ containing
+   the bullet points from ``NEWS.rst``
+#. Check that the new release was built correctly on RTD_, delete the "stable"
+   version and select the new release as default version
```

Comment:
That#s true. Regarding to https://docs.readthedocs.io/en/stable/versions.html it should automatically point to the tag with highest number. I will remove it and check.

Reply:
There is one difference to the current behavior. If you use `stable` as default branch then this will also be displayed as URL. Before we selected the latest version and then if you enter something like `https://sfs.readthedocs.org/` it would e redirected to `https://sfs.readthedocs.org/en/3.2` instead of `https://sfs.readthedocs.org/en/stable`.

But using `stable` it's a lot easier to manage as you don't have to change anything on readthedocs, so we should maybe switch to that.

Reply:
But I just realize that for the case of `https://sfs.readthedocs.org/` it's not a good idea to use stable. As we write that users can hover over the equation links to get the corresponding URL that will always work. That is of cause not the case when `stable` is included into the URL.

So, for now I will stay with the instructions, but remove the `delete the "stable" version` as we only have to select the newest version.

Reply:
> So, for now I will stay with the instructions, but remove the `delete the "stable" version` as we only have to select the newest version.

That's all I meant originally.

I've never used the `stable` name, I don't like it.

The problem was that for some period of time (which seems to be over now), ReadTheDocs automatically re-activated the `stable` thing at each release.
Therefore, we had to disable it at each release. But that nonsense is luckily over now.